xla/service/xla_compile_main.cc

```diff
@@ -68,29 +72,31 @@ StatusOr<std::string> AotCompileCpuExecutable(
   return result;
 }
 
-#if GOOGLE_CUDA
```

Comment:
Is this supposed to be changing the function to be defined where previously it was not in the case of GOOGLE_CUDA and TENSORLOW_USE_ROCM being false?

Reply:
Oh, actually this wasn't intentional.  Will change it back..